# DP: Fix PR ada/114064, 64bit definitions for time type for 32bit archs

--- a/src/gcc/ada/Makefile.rtl
+++ b/src/gcc/ada/Makefile.rtl
@@ -2179,7 +2179,8 @@ ifeq ($(strip $(filter-out mips% linux%,
   s-linux.ads<libgnarl/s-linux__mips.ads \
   s-osinte.adb<libgnarl/s-osinte__posix.adb \
   s-osinte.ads<libgnarl/s-osinte__linux.ads \
-  s-osprim.adb<libgnat/s-osprim__posix.adb \
+  s-osprim.adb<libgnat/s-osprim__posix2008.adb \
+  s-parame.ads<libgnat/s-parame__posix2008.ads \
   s-taprop.adb<libgnarl/s-taprop__linux.adb \
   s-tasinf.ads<libgnarl/s-tasinf__linux.ads \
   s-tasinf.adb<libgnarl/s-tasinf__linux.adb \
@@ -2223,10 +2224,11 @@ ifeq ($(strip $(filter-out powerpc% linu
   s-intman.adb<libgnarl/s-intman__posix.adb \
   s-linux.ads<libgnarl/s-linux.ads \
   s-osinte.adb<libgnarl/s-osinte__posix.adb \
+  s-parame.ads<libgnat/s-parame__posix2008.ads \
   s-tpopsp.adb<libgnarl/s-tpopsp__tls.adb \
   s-mudido.adb<libgnarl/s-mudido__affinity.adb \
   s-osinte.ads<libgnarl/s-osinte__linux.ads \
-  s-osprim.adb<libgnat/s-osprim__posix.adb \
+  s-osprim.adb<libgnat/s-osprim__posix2008.adb \
   s-taprop.adb<libgnarl/s-taprop__linux.adb \
   s-tasinf.ads<libgnarl/s-tasinf__linux.ads \
   s-tasinf.adb<libgnarl/s-tasinf__linux.adb \
@@ -2272,7 +2274,8 @@ ifeq ($(strip $(filter-out arm% linux-gn
   s-linux.ads<libgnarl/s-linux.ads \
   s-osinte.adb<libgnarl/s-osinte__posix.adb \
   s-osinte.ads<libgnarl/s-osinte__linux.ads \
-  s-osprim.adb<libgnat/s-osprim__posix.adb \
+  s-osprim.adb<libgnat/s-osprim__posix2008.adb \
+  s-parame.ads<libgnat/s-parame__posix2008.ads \
   s-taprop.adb<libgnarl/s-taprop__linux.adb \
   s-tasinf.ads<libgnarl/s-tasinf__linux.ads \
   s-tasinf.adb<libgnarl/s-tasinf__linux.adb \
@@ -2385,7 +2388,8 @@ ifeq ($(strip $(filter-out hppa% linux%,
   s-linux.ads<libgnarl/s-linux__hppa.ads \
   s-osinte.adb<libgnarl/s-osinte__posix.adb \
   s-osinte.ads<libgnarl/s-osinte__linux.ads \
-  s-osprim.adb<libgnat/s-osprim__posix.adb \
+  s-osprim.adb<libgnat/s-osprim__posix2008.adb \
+  s-parame.ads<libgnat/s-parame__posix2008.ads \
   s-taprop.adb<libgnarl/s-taprop__linux.adb \
   s-tasinf.ads<libgnarl/s-tasinf__linux.ads \
   s-tasinf.adb<libgnarl/s-tasinf__linux.adb \
@@ -2412,7 +2416,8 @@ ifeq ($(strip $(filter-out m68k% linux%,
   s-linux.ads<libgnarl/s-linux.ads \
   s-osinte.adb<libgnarl/s-osinte__posix.adb \
   s-osinte.ads<libgnarl/s-osinte__linux.ads \
-  s-osprim.adb<libgnat/s-osprim__posix.adb \
+  s-osprim.adb<libgnat/s-osprim__posix2008.adb \
+  s-parame.ads<libgnat/s-parame__posix2008.ads \
   s-taprop.adb<libgnarl/s-taprop__linux.adb \
   s-tasinf.ads<libgnarl/s-tasinf__linux.ads \
   s-tasinf.adb<libgnarl/s-tasinf__linux.adb \
@@ -2439,7 +2444,8 @@ ifeq ($(strip $(filter-out sh4% linux%,$
   s-linux.ads<libgnarl/s-linux.ads \
   s-osinte.adb<libgnarl/s-osinte__posix.adb \
   s-osinte.ads<libgnarl/s-osinte__linux.ads \
-  s-osprim.adb<libgnat/s-osprim__posix.adb \
+  s-osprim.adb<libgnat/s-osprim__posix2008.adb \
+  s-parame.ads<libgnat/s-parame__posix2008.ads \
   s-taprop.adb<libgnarl/s-taprop__linux.adb \
   s-tasinf.ads<libgnarl/s-tasinf__linux.ads \
   s-tasinf.adb<libgnarl/s-tasinf__linux.adb \
